lang/cs/build.props

```diff
@@ -133,4 +134,10 @@ under the License.
       DestinationFiles="@(MySourceFiles->'$(TargetDir)%(Filename)%(Extension)')"
     />
   </Target>
+    <ItemGroup>
+        <DirectoriesToRemove Include="$(IntermediateOutputPath)" />
+     </ItemGroup>
+     <Target Name="RemoveIntermediateDirectories"  AfterTargets="clean">
+        <RemoveDir Directories="@(DirectoriesToRemove)" Condition="!Exists('$(IntermediateOutputPath)\$(RootNamespace).log')"/>
```

Comment:
I'd be in favor of nuking the whole `$(OutputPath)`. That way, the `bin` folder serves the same function as the `target` folder in Maven. We can also put our generated sources there at a later time.